.changeset/cold-tools-admire.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-mud": patch
+---
+
+Updated the react-three template.
```

Comment:
can we describe what changed? this is going into the changelog and it kinda defeats the purpose of this entry if it doesn't actually explain anything

Reply:
also template name is `threejs`